hack/set-span-id.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/vendor/go.opencensus.io/trace/trace.go b/vendor/go.opencensus.io/trace/trace.go
+index 38ead7bf..9e6fe483 100644
+--- a/vendor/go.opencensus.io/trace/trace.go
++++ b/vendor/go.opencensus.io/trace/trace.go
+@@ -261,6 +261,11 @@ func startSpanInternal(name string, hasParent bool, parent SpanContext, remotePa
+ 	return span
```

Comment:
is that until some update?

Reply:
Nope. I just couldn't find a better way to do it. There probably is one, but I wasn't able to find it.

In theory, we will move to some [OpenTelemetry](https://opentelemetry.io/) libraries when they go 1.0, at which point I hope we don't need a similar hack :)